index.html

```diff
@@ -4,15 +4,46 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<!--End-->
     <link rel="stylesheet" href="style.css">
     <title>Setup and Mobile first</title>
 </head>
 <body>
-    <div class="container">
-        <div class="box">B 1</div>
-        <div class="box one">B 2</div>
-        <div class="box">B 3</div>
-        <div class="box">B 4</div>
+<header>
+    <div class="top-logo">
+        <a href="#">Abbas</a>
     </div>
+    <nav class="Navbar">
+        <div class="menu-icon">
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+        </div>
+    </nav>
+</header>
+    <main>
+        <div class="informative-top">
+            <div class="top-heading">
+                <h1>Hello, My name is Abbas Glad to see you!</h1>
+            </div>
+            <div class="heading-description">
+                <p>I'm a web developer! I can help you to create a Product or Portfolio website for you. Check out my website and contact me if you find it great.</p> 
+            </div>
+            <div class="touch-links">
+                <h3><a href="#">LET'S CONNECT</a></h3>
+                <div class="social-bar">
```

Comment:
- [ ] Like the previous reviewer mentioned, to improve the page accessibility for users (blind) who depend on using the keyboard `tab` key to navigate the page, it is best practice to wrap links like these social media icons in a `ul` > `li`, and then `a` tag.  I suggest that you use the ul list tag for the social media links and wrap each of the <a> tags inside <li> list tags.